doc/specs/permissions.md

```diff
@@ -0,0 +1,77 @@
+# Permissions
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+The site requires certain pages and abilities to be limited to a sub set of users. A user can both be granted extra permissions and have certain permissions removed. A user typically gains extra permissions when transcending classes but can also be granted permissions explicitly. 
+
+## Business rules
+
+ - If a user lacks the permission to access a page then they must not be able to view it and taken to an error page.
+ - If a user lacks the permission to perform a specific action they must not be able to perform it. 
+ - Any change to a user's permissions from an administrative user must be logged in the user's log, indicating reason for the change and the permissions which were added or removed. 
+
+## Use cases
+
+### 1.0 Grant a user permission
+
+**Primary Actor:** An administrator.
```

Comment:
Primary actor makes me think that there is such a thing as a 'secondary actor'. Just a nitpick here.